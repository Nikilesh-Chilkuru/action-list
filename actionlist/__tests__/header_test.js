import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../header';

test('renders correctly', () => {
  const tree = renderer.create(
    <Header />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
