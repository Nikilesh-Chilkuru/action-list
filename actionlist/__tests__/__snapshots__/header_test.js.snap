exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#7a1705",
      "borderTopColor": "#990000",
      "borderTopWidth": 5,
      "flexDirection": "row",
    }
  }>
  <Image
    source={1}
    style={
      Object {
        "height": 50,
        "marginLeft": 10,
        "width": 43,
      }
    } />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "alignItems": "center",
        "color": "#fff",
        "flex": 1,
        "fontFamily": "BentonSansBold, Arial, sans-serif",
        "fontSize": 24,
        "fontWeight": "bold",
        "justifyContent": "center",
        "marginLeft": 10,
        "marginTop": 7,
      }
    }>
    IU Action List
  </Text>
  <Image
    source={1}
    style={
      Object {
        "height": 16,
        "margin": 15,
        "width": 16,
      }
    } />
  <Image
    source={1}
    style={
      Object {
        "height": 16,
        "justifyContent": "flex-end",
        "margin": 15,
        "marginRight": 20,
        "width": 16,
      }
    } />
</View>
`;
